src/dmir.rs

```diff
@@ -558,7 +559,63 @@ pub fn decode_byond_bytecode(nodes: Vec<Node<DebugData>>, proc: Proc) -> Result<
                         decode_set_var(&var, &mut irs);
                     }
                     Instruction::IsNull => {
-                        irs.push(DMIR::IsNull)
```

Comment:
Why some operations produce DMValue as result, while other changes test flag, is it intended?

Reply:
Because byond opcodes has different behavior, isnull, isnum, istext dont change test flag